Calculator SwiftUIUITests/Tests/CalculatorTest.swift

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/DawidBastek/UITestsForCalculator/blob/bc65a75b1eca9a594cdbdc822924004c9c4d0549/Calculator SwiftUIUITests/Tests/CalculatorTest.swift#L1"></span>File name should match a type or extension declared in the file (if any).<br />
<code>file_name</code> <code>CalculatorTest.swift:1</code></td>
    </tr>
  </tbody>
</table>